src/cli_plugin/list/list.js

```diff
@@ -0,0 +1,14 @@
+import { statSync, readdirSync } from 'fs';
+import { join } from 'path';
+
+export default function list(settings, logger) {
+  readdirSync(settings.pluginDir)
+  .forEach((filename) => {
+    const stat = statSync(join(settings.pluginDir, filename));
+
+    if (stat.isDirectory() && filename[0] !== '.') {
+      logger.log(filename);
+    }
+  });
+  logger.log('');
```

Comment:
Was this intentional? If so, can you add a comment here to say what it's for because it looks like an accident.